scoring_engine/models/team.py

```diff
@@ -30,7 +32,7 @@ def current_score(self):
 
     @property
     def place(self):
-        sorted_blue_teams = sorted(self.blue_teams, key=lambda team: team.current_score, reverse=True)
+        sorted_blue_teams = sorted(Team.get_all_blue_teams(), key=lambda team: team.current_score, reverse=True)
```

Comment:
This isn't related to the cache, just so that we can remove the blue_teams property.